src/components/common/Dialogs/TransferFundsDialog/TransferFundsDialogForm.tsx

```diff
@@ -54,10 +61,13 @@ const MSG = defineMessages({
 const TransferFundsDialogForm = ({ back, colony }: ActionDialogProps) => {
   const { wallet } = useAppContext();
   const {
-    getValues,
     formState: { isSubmitting, isValid },
+    watch,
   } = useFormContext();
-  const values = getValues();
+  const fromDomainId = watch('fromDomainId');
+  const toDomainId = watch('toDomainId');
+  const amount = watch('amount');
+  const tokenAddress = watch('tokenAddress');
```

Comment:
You can also use `watch()` here and then `const { fromDomainId, toDomainId, amount, tokenAddress } = watch()`